tests/libvmi/factory.go

```diff
@@ -34,6 +34,18 @@ const (
 // NewFedora instantiates a new Fedora based VMI configuration,
 // building its extra properties based on the specified With* options.
 func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	return newFedora(cd.ContainerDiskFedora, opts...)
+}
+
+// NewCustomFedora instantiates a new Fedora based VMI configuration,
+// building its extra properties based on the specified With* options.
+func NewSriovFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	return newFedora(cd.ContainerDiskFedoraSRIOVLane, opts...)
+}
+
+// NewFedora instantiates a new Fedora based VMI configuration,
```

Comment:
NewFedora -> newFedora

Please add that the flavor of the image is provided (that is the main difference from the public one).

Reply:
Done